pages/common/perl.md

```diff
@@ -0,0 +1,23 @@
+# perl
+
+> The Perl 5 language interpreter 'CLI'.
```

Comment:
I got it from 'php.md' which I assumed that it should be included, as they both are CLI interpreters. I just find similar commands to figure out the style of the page I'm writing. It can delete that if you want.